src/EditorFeatures/Core/BackgroundWorkIndicator/BackgroundWorkIndicatorContext.cs

```diff
@@ -5,14 +5,9 @@
 using System;
```

Comment:
this file is effectively a rewrite.  I would look at the SxS view, not hte inline view.  And really just pay attention to how the new code is written.